### PR TITLE
TW-485: Sort tokens by dollar value

### PR DESCRIPTION
**Pull request by alex-tsx**

### CODE REVIEW

**Review by keshan3262 — CHANGES_REQUESTED** [review]

**Review by keshan3262 — DISMISSED** [review]

**Review by herkoss — APPROVED** [review]
Well done!)

**Review by keshan3262 — APPROVED** [review]

File: src/hooks/use-filtered-assets-list.hook.ts [hunk 1]
```diff
@@ -1,45 +1,72 @@
...
+    return sortByDollarValueDecrease ? applySortByDollarValueDecrease(result) : result;
+  }, [searchValue, sourceArray, sortByDollarValueDecrease]);
+
+  const filteredAssetsList = useMemo(() => {
+    if (!isDefined(leadingAsset)) {
+      return searchedAssetsList;
+    }
+
+    if (leadingAssetIsFilterable) {
+      if (filterZeroBalances && !isNonZeroBalance(leadingAsset)) {
```
Comment (herkoss) [inline src/hooks/use-filtered-assets-list.hook.ts:43]: Maybe we need `isZeroBalance` helper?

**Review by herkoss — APPROVED** [review]
GJ!

**Review by keshan3262 — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
build version: 1.14_565б android done

**Review by NovakVlad2424 — APPROVED** [review]
build version: 1.14_565 ios done

**Pull request merged**